pyproject.toml

```diff
@@ -124,7 +124,7 @@ python-dateutil = ['dateutil']
 pytest-plone = ['pytest', 'zope.pytestlayer', 'plone.testing', 'plone.app.testing']
 'Products.CMFPlone' = ['Acquisition', 'BTrees', 'Missing', 'OFS', 'plone.app.contenttypes', 'plone.app.layout', 'plone.app.linkintegrity', 'plone.app.redirector', 'plone.app.registry', 'plone.app.textfield', 'plone.app.vocabularies', 'plone.app.z3cform', 'plone.autoform', 'plone.base', 'plone.behavior', 'plone.dexterity', 'plone.indexer', 'plone.namedfile', 'plone.protect', 'plone.registry', 'plone.rfc822', 'plone.schema', 'plone.supermodel', 'Products.BTreeFolder2', 'Products.CMFCore', 'Products.Five', 'Products.GenericSetup', 'Products.SiteErrorLog', 'transaction', 'z3c.form', 'z3c.relationfield', 'zope.component', 'zope.i18n', 'zope.i18nmessageid', 'zope.interface', 'zope.lifecycleevent', 'zope.publisher', 'zope.schema', 'Zope']
 'plone.restapi' = ['plone.rest']
-ignore-packages = ['collective.folderishtypes', 'plone.app.caching', 'plone.app.multilingual', 'requests', 'responses', 'scripts.utils', 'zest.releaser', 'zestreleaser.towncrier']
+ignore-packages = ['plone.app.caching', 'plone.app.multilingual', 'requests', 'responses', 'scripts.utils']
```

Comment:
You need to change `.meta.toml`:

https://github.com/plone/plone.volto/blob/af295193ef1c88367938b84b59839ec88e0fe30b/.meta.toml#L16

And run the `plone/meta` script in this package.